applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/Kconfig

```diff
@@ -211,6 +211,12 @@ config BT_AUDIO_BITRATE_BROADCAST_SRC
 config BT_AUDIO_SCAN_DELEGATOR
 	bool "Enable scan delegator"
 	depends on TRANSPORT_BIS
+	select BT_CAP_ACCEPTOR
+	select BT_CSIP_SET_MEMBER
+	select BT_CAP_ACCEPTOR_SET_MEMBER
+	select BT_GAP_PERIPHERAL_PREF_PARAMS
+	select BT_VCP_VOL_REND
+	select BT_PER_ADV_SYNC_TRANSFER_RECEIVER
 	default n
```

Comment:
```suggestion
```

Reply:
Is that because `default n` is always implied if nothing is specified?

Reply:
yes